common/lib/xmodule/xmodule/tabs.py

```diff
@@ -911,23 +309,16 @@ def validate_tabs(cls, tabs):
         if len(tabs) < 2:
             raise InvalidTabsException("Expected at least two tabs.  tabs: '{0}'".format(tabs))
 
-        if tabs[0].get('type') != CoursewareTab.type:
+        if tabs[0].get('type') != 'courseware':
             raise InvalidTabsException(
                 "Expected first tab to have type 'courseware'.  tabs: '{0}'".format(tabs))
 
-        if tabs[1].get('type') != CourseInfoTab.type:
+        if tabs[1].get('type') != 'course_info':
             raise InvalidTabsException(
                 "Expected second tab to have type 'course_info'.  tabs: '{0}'".format(tabs))
 
         # the following tabs should appear only once
-        for tab_type in [
-                CoursewareTab.type,
-                CourseInfoTab.type,
-                NotesTab.type,
-                TextbookTabs.type,
-                PDFTextbookTabs.type,
-                HtmlTextbookTabs.type,
-                EdxNotesTab.type]:
+        for tab_type in ['courseware', 'course_info', 'notes', 'textbooks', 'pdf_textbooks', 'html_textbooks']:
```

Comment:
I think we should have a field on ``CourseViewType`` to indicate the number of times it can be shown, and then we won't need this special-cased list. Maybe ``max_occurrences``?